tests/unit/test_rt0.py

```diff
@@ -1230,7 +1230,7 @@ def test_rt0_2d_simplex_surf(self):
 
     def test_convergence_rt0_2d_iso_simplex(self):
 
-        a = 8 * np.pi ** 2
+        a = 8 * np.pi**2
```

Comment:
I guess this and severeal other random changes in the tests is caused by running black. As a rule, I would prefer if we do so in dedicated PRs since this will make the review much simpler: We are not allowed to argue with black so that PR is accepted by default, while for standard PRs, the reviewer will not have to filter away random formatting edits.